docs/faq/introduction-to-augur.md

```diff
@@ -0,0 +1,27 @@
+# Augur: Nextstrain's Bioinformatics Toolkit
```

Comment:
I was surprised to find the "introduction to Augur" in the FAQ section of the docs when I think of this as the content for the front page of the Augur docs.

Does this content need to be here for something about RTD subprojects to work?

Reply:
This was migrated from https://nextstrain.org/docs/bioinformatics/introduction-to-augur but I don't think it makes a ton of sense in this section either. It could be merged with the front page of the Augur docs since it serves the same purpose as you point out. I'm glad to quickly merge their collective content into the augur docs main index page unless you prefer to do it?

Reply:
I prefer the content of the current Augur main index page, so I would consider deleting this intro to augur page unless @jameshadfield feels strongly about keeping any of its content.

Reply:
Sounds good, I just removed it, since most of its content felt like it was already addressed in the existing index page. I did update the warning that linked back to docs.nextstrain.org:
![Screen Shot 2020-11-04 at 11 40 49 AM](https://user-images.githubusercontent.com/12140437/98160271-cb4fdf80-1e92-11eb-95e6-ada8a7754eac.png)

To provide more context (with a few links) about the difference between the subproject and the main docs.nextstrain.org project. I also made it a note instead of a warning since that seemed more appropriate. I also added a link to the augur github repo which was in the deleted index page, since I didn't see one in the existing index page and it seemed helpful:

![Screen Shot 2020-11-04 at 11 40 01 AM](https://user-images.githubusercontent.com/12140437/98160412-05b97c80-1e93-11eb-9057-b72cbb194023.png)